src/app.js

```diff
@@ -2,24 +2,41 @@
 
 require('dotenv').config();
 
+const pg = require('pg');
+const sql = require('./sql');
```

Comment:
what's the organization schema for these imports?

Reply:
We did a refactor today and reorganized our imports.  Third-party imports are above our module imports, and each group is in alphabetical order.